python/src/grid/grid_range.hpp

```diff
@@ -34,6 +34,7 @@ void add_grid_range(py::module_& m) {
         .def("n_tiles", py::overload_cast<size_t>(&gr_t::n_tiles, py::const_),
              py::arg("gridpt_idx"))
         .def("n_tiles", py::overload_cast<>(&gr_t::n_tiles, py::const_))
+        .def_readonly("cum_n_tiles", &gr_t::cum_n_tiles_)
```

Comment:
Might be too much to ask, but while you're at it, can I ask you to change all of these getter functions as `.def_readonly` like this? That was on my TODO but now might be a good time to just change all of them.

Reply:
Yuppp